### DISCUSSION TITLE
how to show custom fields in online users panel

### DISCUSSION THREAD

**Discussion by salsero72** [LiveHelperChat/livehelperchat#1850] [General]
Hello
How to show custom fields in online users panel?

I've read you can add custom fields but the manual explains how to show them in chat lists only. Not in online users panel
[https://doc.livehelperchat.com/docs/custom-fields-and-prefill/](https://doc.livehelperchat.com/docs/custom-fields-and-prefill/)

**Comment by remdex** [comment]
I suppose you missed checkbox, show in online visitors list...
